app/lib/project/MetapolatorProject.js

```diff
@@ -626,6 +627,10 @@ define([
         }
     };
 
+    _p.saveAs = function(blob, filename) {
```

Comment:
Even if this would be a good place for the `saveAs` method, I think the implementation here should be done so:

```js
_p.saveAs = saveAs;
```